src/stan-users-guide/odes.Rmd

```diff
@@ -354,25 +354,26 @@ indiscriminately stepping over an important feature of the solution.
 
 ### Tolerance {-}
 
-The relative and absolute tolerance control the accuracy of the
+The relative tolerance RTOL and absolute tolerance ATOL control the accuracy of the
 numerical solution. Specifically, when solving an ODE with unknowns
-$y=(y_1,\dots,y_n)^T$, the
-solver controls estimated local error $e=(e_1,\dots,e_n)^T$ at
-certain step through its weighted root-mean-square norm $\|\cdot\|$
+$y=(y_1,\dots,y_n)^T$, at every step the
+solver controls estimated local error $e=(e_1,\dots,e_n)^T$ through its weighted root-mean-square norm
```

Comment:
The quantity below sums over states -- do we control local error as a vector or do we just control the quantity `|e|`? Clarify this (or tell me I'm misunderstanding something).

Reply:
We control `|e|`.

Reply:
Okay cool make it like that in the text then.

Reply:
Now I see what you mean. We're at natural language limit here. We do **control** `e`, by limiting its norm. The wording here is the same as in CVODE user guide (p.24) and I'm assuming it's generally accepted in numerical analysis world.

Reply:
Oh, I didn't read the part after the dollars. Makes sense now.